### ISSUE TITLE
Add projects on the website

### ISSUE THREAD

**Issue by v-thakkar** [rails-girls-summer-of-code/summer-of-code#439]
We have been asked many times by prospective applicants and project mentors about the projects which have participated in the past. As of now, we don't really have any such information on the website. Adding a separate section for the projects will help in 2 ways:

- It will give an idea to prospective applicants, sponsors and project mentors about what kind of projects have participated in the past and can help them decide about their participation in our next edition. 
- As of now, our progarm's name do not give an impression of being a langauge agnostic program. This information will encourage more projects to participate in the program.
 
I would also like to open this issue as a discussion point to see how we would like this to see this on our website from the design point of view. I have 2 ideas but I'm open for more ideas as well.

1. One way to do this is to have separate github pages for each year's projects which can apparaently have details about the projects and teams who worked on those projects in the respective years. Main projects page can be updated after the selection of projects in each year. And previous year's pages can then be moved to archive. 

2. Another way would be to have just one github page with each year's project list and teams participated.

**Comment by inescoelho** [comment]
IMO, we don't need a project page. We need a page with information about each edition of RGSOC, including the projects that were selected/participating each year. That way we convey the message you wabt to portrait without focusing so much on the projects, which is not the goal of RGSoC.

**Comment by v-thakkar** [comment]
I agree with your approach @inescoelho. When I opened the issue, I just wanted to add this before we open the call for projects as lot of projects need to look at which projects have participated in the past [program name issue again]. But now that re-branding work is going in parallel and in no way this seems to be implemented before that, my vote definitely goes for adding information for each RGSoC edition - teams, projects etc. But I would still like to see information about the projects accepted in each edition instead of the projects which participated in the program at the end.
